### ISSUE TITLE
Calling increment/decrement with tags, callback and no value 

### ISSUE THREAD

**Issue by aheuermann** [bdeitte/hot-shots#139]
I noticed that the following call results in an invalid metric format.

```js
hotShots.increment('some.metric', { tagName: 'tags' }, callback);
```
I think the issue is due to the [arguments length check here](https://github.com/brightcove/hot-shots/blame/master/lib/statsFunctions.js#L103).

**Comment by bdeitte** [comment]
Yep the way that increment and other functions deal with missing args is... unusual.  But it's been this way for a long time and is a bit tough to consider changing for legacy reasons.  If someone sees ways we can make this better that looks safe, and tests cover it, I'm good with it.

**Comment by bdeitte** [comment]
Fixed in 14.0.0
